compiler/sc1.cpp

```diff
@@ -1359,20 +1359,14 @@ static void dumplits(void)
  */
 static void dumpzero(int count)
 {
-  int i;
-
   if (sc_status==statSKIP || count<=0)
     return;
   assert(curseg==2);
-  defstorage();
-  i=0;
-  while (count-- > 0) {
-    outval(0, FALSE);
-    i=(i+1) % 16;
-    stgwrite((i==0 || count==0) ? "\n" : " ");
-    if (i==0 && count>0)
-      defstorage();
-  } /* while */
+  
+  stgwrite("dumpdup ");

```

Comment:
This change seems good - could you call it "dumpfill" or something instead? "dup" usually means something else in VMs and there's precedent for "fill" here.

Reply:
Sure, I wasn't keen on the naming anyway. I chose that naming because of ASM `DUP(value)`.
Do you want me to amend or create a new commit?

Reply:
Up to you - I'll squash it in the end.

Reply:
Out of curiosity - what's the usual meaning of "dup" in VMs?